### DISCUSSION TITLE
Send don't work

### DISCUSSION THREAD

**Discussion by Dragod** [dani-garcia/vaultwarden#3138] [General]
I tried to use a send in my self-hosted instance but it doesn't work. I get this error:

Refused to connect because it violates the following Content Security Policy Directive. connect-src 'self'

How can I fix that?

![msedge_K1NjuyRNii](https://user-images.githubusercontent.com/4368630/212272020-19fc6493-189b-451e-9cbe-c5245bf5b805.png)

**Comment by BlackDex** [comment]
You can fix that by correcting your DOMAIN config.
That seems to be not configured correctly.

It needs to match.

**Reply by Dragod** [reply]
Excuse my ignorance, where would I do that? Thanks a million.

**Reply by BlackDex** [reply]
You should configure the `DOMAIN` variable.
I'm not sure how you started your Docker Container but you should configure it there.
Also, you are running an older version of Vaultwarden which has issue with the current clients versions of Bitwarden.

**Reply by Dragod** [reply]
I use portainer with a compose stack. Did find out how to update to latest version, but still not understanding how to update that domain variable.

**Reply by BlackDex** [reply]
Just the first item i found on the internetz.
https://www.techrepublic.com/article/add-environment-variables-container-portainer/

**Reply by Dragod** [reply]
Fixed it thanks. I had the port number at the end which was causing the issue... thanks a lot
